theano/gpuarray/subtensor.py

```diff
@@ -469,6 +466,7 @@ class GpuAdvancedIncSubtensor1(Op):
     Implement AdvancedIncSubtensor1 on the gpu.
 
     """
+    _f16_ok = True
```

Comment:
Add some basic test that it work. I think the C code don't support it. It don't pass the new flag. If this is not needed, can you add a comment when creating GpuElemwise_new() to tell this?